Dockerfile.build

```diff
@@ -2,9 +2,20 @@ FROM archlinux:base-devel
 
 ENV RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
-    PATH=/usr/local/cargo/bin:$PATH
+    PATH=/usr/local/cargo/bin:$PATH \
+    POETRY_HOME=/opt/poetry \
+    CLIENT_CLI_DIR=/__w/iroha/iroha/target/debug
 
-RUN pacman -Syu rustup mold musl rust-musl openssl libgit2 git docker docker-buildx docker-compose --noconfirm
+ENV PATH=$POETRY_HOME/bin:$PATH
+
+RUN pacman -Syu rustup mold musl rust-musl openssl libgit2 \
+    git docker docker-buildx docker-compose \
+    python python-pip --noconfirm --disable-download-timeout && \
+    curl -sSL https://install.python-poetry.org | python3 -
+
+COPY . .
+WORKDIR client_cli/pytests/
```

Comment:
this is going to affect later runs? I see `git config --global --add safe.directory .` at least will be affected

Reply:
> this is going to affect later runs? I see `git config --global --add safe.directory .` at least will be affected

If I understood Vasiliy correctly, he thinks that if we don't copy everything, but make `COPY /client_cli/pytests/pyproject.toml /client_cli/pytests/` as we have already done now, it will be enough.